olive/passes/onnx/quantization.py

```diff
@@ -611,23 +618,65 @@ def _default_config(cls, accelerator_spec: AcceleratorSpec) -> Dict[str, PassCon
             "algorithm": PassConfigParam(
                 type_=str,
                 default_value=None,
-                description="Available from onnxruntime>=1.17.0 "
-                "Algorithm config for quantization, can be 'DEFAULT', 'HQQ' or 'RTN', 'GPTQ' "
-                "(default DEFAULT when not set). For 4b quantize a model with RTN or GPTQ algorithm. "
-                "Please refer to https://github.com/intel/neural-compressor/blob/master/docs/source/quantization_weight_only.md "  # noqa: E501
+                description="'RTN' and 'GPTQ' are available from onnxruntime>=1.17.0 "
+                "'DEFAULT', 'HQQ' are available from onnxruntime>=1.18.0 (default DEFAULT when not set). "
+                "For 4b quantize a model with RTN or GPTQ algorithm. Please refer to "
+                "https://github.com/intel/neural-compressor/blob/master/docs/source/quantization_weight_only.md "
                 "for more details on weight only quantization using Intel® Neural Compressor. "
                 "For HQQ, please refer to onnxruntime for more details: "
                 "https://github.com/microsoft/onnxruntime/blob/main/onnxruntime/python/tools/quantization/matmul_4bits_quantizer.py#L102C1-L126C25",
             ),
+            "weight_only_quant_configs": PassConfigParam(
+                type_=dict,
+                default_value=None,
+                description="""Available from onnxruntime>=1.17.0
+                The config is binding to the algorithm with following map:
+                1. "algorithm" is "DEFAULT", by default, the weight_only_quant_configs is:
+                    "weight_only_quant_configs": {
+                        "block_size": 128,
+                        "is_symmetric": False,
+                        "accuracy_level": None
+                    }
+                    https://github.com/microsoft/onnxruntime/blob/main/onnxruntime/python/tools/quantization/matmul_4bits_quantizer.py#L129C1-L140C45
```

Comment:
If there is any doc link rather than main branch code link? Cause this link will point to wrong line if the ort code changes. Same as below. Or if there is no doc, we can use a permanent branch link rather main branch.

Reply:
Good catch. Updated with commit changes.